### DISCUSSION TITLE
Fixed: README incorrectly says "Please use GitHub for bug reports"

### DISCUSSION THREAD

**Discussion by eli-schwartz** [samboy/MaraDNS#123] [General]
This is horribly confusing, since Github Issues are not enabled, and in desperation I spent about half an hour checking all mirror sites to see if they have issues enabled -- codeberg and gitlab do, sourcehut doesn't -- and wondering why the enabled issue trackers have no bug reports. Due to reasons even I don't understand, while I was in the middle of writing a bug report on gitlab, I noticed this hidden text on the maradns website itself: "Please file all bug reports at Github, using the discussions tab."

This is not idiomatic usage of Github Discussions, so you should NOT assume that people are going to guess that's where a bug report should go without saying so.

**Comment by samboy** [comment]
Fixed in commit 2fa74606086958b154d76c2c21094378add1d2bb

**Comment by samboy** [comment]
The reason why I turned off GitHub issues for MaraDNS:

* Too many issues were feature requests
* GitHub would list the number of open issues when people opened the GitHub webpage; I would work around this by marking issues as “Won’t do without getting paid” but other issues would be opened. 
* One issue was someone asking for hand-holding support. When I told them no, they got abusive with me.
